ddtrace/debugging/_encoding.py

```diff
@@ -431,10 +430,10 @@ def _snapshot_v2(snapshot):
     }
 
 
-def _logger_v2(snapshot):
-    # type: (Snapshot) -> Dict[str, Any]
-    thread = snapshot.thread
-    code = snapshot.frame.f_code
+def _logger_v2(thread, frame):
```

Comment:
I think we can perhaps rename the argument but keep the signature of this function? It looks like both snapshots and logs have the thread and frame subfields.